util/econftool.c

```diff
@@ -100,18 +106,13 @@ static int nftw_remove(const char *path, const struct stat *sb, int flag, struct
 static int generate_tmp_file(char *tmp_name)
 {
     int filedes;
-    char name_template[21] = "/tmp/econftool-XXXXXX";
-    filedes = mkstemp(name_template);
```

Comment:
This opens the file, so it has to be closed here to avoid leaking the file descriptor